dockermon.py

```diff
@@ -74,7 +74,7 @@ def watch(callback, url=default_sock_url):
     sock = connect(url)
 
     with closing(sock):
-        sock.sendall(b'GET /events HTTP/1.1\n\n')
+        sock.sendall(b'GET /events HTTP/1.1\nHost: foo\n\n')
```

Comment:
I prefer to use the real host name if it's a TCP connection and `localhost` if it's IPC.